aws/README.md

```diff
@@ -0,0 +1,63 @@
+# Deploy to AWS (Beta)
+
+> **Note:** The AWS deployment template is still in **BETA** phase. You may encounter failures during the deployment. In that case, please let us know the issue.
+
+This template will create a fully-working IPSec/L2TP VPN server on AWS (Amazon Web Service). Please make sure to check the [pricing details](https://aws.amazon.com/ec2/pricing/on-demand/) of Virtual Machine on EC2 before starting the launch sequence.
+
+You can also use `t2.micro` instance as your server for your deployment, which is free of charge within the first year since your AWS account is registered. For more information on AWS free usage tier, go to [this page](https://aws.amazon.com/free/).
+
+## Available Customization Parameters:
+
+- AWS EC2 Instance Type
+- OS for your VPN Server (Ubuntu16.04, Ubuntu18.04 or Debian9-stretch)
+- Your VPN username
+- Your VPN password
+- IPSec PSK (pre-shared key)
+
+> When choosing your username and password, do not enter special characters like `" ' \`.
+
+Make sure to do this with an **AWS ROOT ACCOUNT** or an **IAM ACCOUNT** with **ADMINISTRATION PRVILEGE**. AWS Cloudformation requires the template file to be store on AWS S3, while a [copy](./cloudformation-template-ipsec) is available on Github. If you want to make any contributions to this template, make sure to contact me so that I can update the template on S3.
+
+If you are confused about the instance types available, go to the [FAQ](#faqs) section and check out **"Instance Type Selection"**.
+
+Click the icon below to initiate the launching sequence.
+
+<a href="https://console.aws.amazon.com/cloudformation/home#/stacks/new?templateURL=https://vpn-tutorial-template.s3.ca-central-1.amazonaws.com/cloudformation-template-ipsec"><img src="../docs/images/cloudformation-launch-stack-button.png" alt="Deploy to AWS" height="60px"></a>
+
+Make sure the deployment is successful before going to [Next Step: Configure VPN Clients](https://git.io/vpnclients).
+
+# FAQs
+<details>
+<summary>
+Instance Type Selection
+</summary>
+
+I have made only a few general-purpose, x64-based instance types available. That's because not all instance types are available in all AWS Regions. Even some of the instance types available in the template are not available in certain AWS regions. So be careful which instance type to choose. The figure below shows the number of regions where each of the selectable instance type is available. 
+
+![](instance-type-sheet.png)
+
+A spreadsheet that includes the raw instance data across all AWS regions is available [here](https://vpn-tutorial-template.s3.ca-central-1.amazonaws.com/Analysis+on+Regional+Availability+of+Selected+Instances+Types+on+AWS+EC2.xlsx). 
```

Comment:
This note seems unnecessary for the purpose of using this template.

Reply:
My concern is that people might get confused by the fact that only few instance types are provided. And they also may encounter error because the selected instance type is not available in the selected region.